lib/Constants.js

```diff
@@ -370,5 +370,244 @@ module.exports = {
   },
   commandClass (cmd) {
     return this._commandClassMap[cmd] || 'unknownClass_' + cmd
+  },
+  _genericDeviceClassMap: {
+    // https://github.com/home-assistant/core/blob/dev/homeassistant/components/zwave/const.py#L196
+    // 0x00: specific_type_not_used // Available in all Generic types
```

Comment:
Could you also add a reference to: https://github.com/OpenZWave/open-zwave/blob/master/config/device_classes.xml

Reply:
Ah nice I imagined that this file existed somewhere in openzwave but for some reason I missed when looking for it! Thanks for pointing it out!

Reply:
@p4p3r I just don't know how I could have missed it till now. From when I started developing hass discovery I always had problems to identify the device

Reply:
Fixed in 7456e4d7b8dfc1197bcca9e247d2712019cc8265